src/dune_rules/compilation_context.mli

```diff
@@ -34,6 +34,7 @@ val create :
   -> ?stdlib:Ocaml_stdlib.t
   -> js_of_ocaml:Js_of_ocaml.In_context.t option
   -> package:Package.t option
+  -> ?melange:Path.Build.t * string
```

Comment:
What's the string argument?

Reply:
The `target` (output folder):

https://github.com/ocaml/dune/blob/270b83e480db47432100fbae7035570bcfed51f4/src/dune_rules/compilation_context.ml#L185

Is there an existing type for this kind of data that I should use for decoding?

Reply:
I don't think so, but I think this argument should turn into a record.

Reply:
Fixed in https://github.com/ocaml/dune/pull/6349/commits/295dd775a5f9aa251807e579e51db265a83ea37a (this made me realize I had added some `Melange.In_context` but I was not using it 🙈 ).